pkg/auth/auth.go

```diff
@@ -176,12 +177,18 @@ func (u *UserAuth) isAllowed(verb string, itemsToCheck []resource) ([]Action, er
 			}
 		}
 		if !allowed {
-			rejectedActions = append(rejectedActions, Action{
+			rejectedAction := Action{
 				APIVersion: i.APIVersion,
 				Resource:   resource,
-				Namespace:  i.Namespace,
```

Comment:
what has been the value of namespace when the resource was actually cluster-wide?

If it is nil or we ignore it in the UI, your code could be potentially reduced to

```
if !namespaced {
  rejectedAction.ClusterWide = true
}
```